src/png.imageio/png_pvt.h

```diff
@@ -684,6 +684,13 @@ write_info(png_structp& sp, png_infop& ip, int& color_type, ImageSpec& spec,
                      (png_uint_32)(yres * scale), unittype);
     }
 
+#ifdef PNG_eXIf_SUPPORTED
+    std::vector<char> exifBlob;
+    encode_exif(spec, exifBlob, endian::big);
+    png_set_eXIf_1(sp, ip, static_cast<png_uint_32>(exifBlob.size()),
```

Comment:
Should we also be using png_get_eXIf functions for the reading, or is what we have sufficient?

Reply:
Looks like this is already the case: https://github.com/OpenImageIO/oiio/blob/master/src/png.imageio/png_pvt.h#L326-L336

Reply:
Indeed, OIIO seems to read Exif from PNG from the eXIf chunk and from embedded text (unofficial convention from before the eXIf chunk existed).